jobs/doppler/templates/doppler_ctl.erb

```diff
@@ -31,8 +31,12 @@ case $1 in
 
     <% p("doppler.debug") == true ? debug_string = "--debug " : debug_string = "" %>
 
+    <% if p('doppler.log_to_syslog') %>
+    tee_output_to_sys_log "$LOG_DIR" doppler stdout stderr
```

Comment:
Yes, but it has a different name. 

Above it uses "doppler_ctl" as basename, "err" for stderr suffix and "" for stdout suffix.

Basically it generates the files:
 * Line 20: `dopper_ctl.log` and `doppler_ctl.err.log`
 * Line 35:  `dopper.stdout.log` and `doppler.stderr.log` (as it was before)

I did not want to change the behaviour and the name of the files.